azure-pipelines.release.web-components.yml

```diff
@@ -75,7 +75,7 @@ extends:
                 displayName: Build, [test], Lint
 
               - script: |
-                  yarn publish:beachball -n $(npmToken) --config scripts/beachball/src/release-web-components.config.js
+                  yarn publish:beachball -n $(npmToken) --config scripts/beachball/src/release-web-components.config.js 'release: applying package updates - web-components'
```

Comment:
Is this correct? Isn't missing the `--message` to specify the param?

Reply:
good catch, yup typo ty!